slices/libmagic-mgc.yaml

```
@@ -0,0 +1,13 @@
+package: libmagic-mgc
+
+essential:
+  - libmagic-mgc_copyright
+
+slices:
+  data:
+    contents:
+      /usr/lib/file/magic.mgc:
```

```suggestion
      /usr/lib/file/magic.mgc:
      /usr/share/file/magic.mgc:
      /usr/share/misc/magic.mgc:
```

Should we also add these symlinks here?